### PR TITLE
Add info to README/templates on float specifiers and representative images

### PR DESCRIPTION
**Pull request by davidlmobley**
Partially addresses #3 

@dwsideriusNIST or @mrshirts can you please review? Makes updates to templates and README.md to mention representative images, give suggestions on float specifiers.

### CODE REVIEW

**Review by dwsideriusNIST — CHANGES_REQUESTED** [review]
I will approve this PR after my PR (#79) is merged into the improve_instructions branch

**Comment by davidlmobley** [comment]
@dwsideriusNIST merged yours. Want to go ahead and merge this one?

**Review by dwsideriusNIST — APPROVED** [review]

**Comment by dwsideriusNIST** [comment]
Go ahead with the merge.

**Pull request merged**
